src/fakeprinter/fakeprinter/__init__.py

```diff
@@ -18,6 +19,9 @@
 STARTED = datetime.now()
 app = Flask(__name__)
 
+app.wsgi_app = ThrottlingMiddleware(float(os.environ.get('THROTTLE', 0), app.wsgi_app)
```

Comment:
Missing bracket -> invalid syntax error

Reply:
fixed in 497638cc030d6e140d703934e0feb7120e1154b5